### PR TITLE
Add explicit reference to did rubric

### PR DESCRIPTION
**Pull request by mprorock**
This can serve as a basis to then link to other sections of the rubric throughout the implementation guide

***
<a href="https://pr-preview.s3.amazonaws.com/mesur-io/did-imp-guide/pull/31.html" title="Last updated on Sep 14, 2021, 8:09 PM UTC (0f0b9cb)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-imp-guide/31/967cb7d...mesur-io:0f0b9cb.html" title="Last updated on Sep 14, 2021, 8:09 PM UTC (0f0b9cb)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -209,6 +209,13 @@ <h2>Methods</h2>
       and its privacy and security properties for comparison to other methods.
     </p>
 
+    <p class="advisement">
+      Developers are encouraged to utilize the [[DID-RUBRIC]] as early as poosible
+      in the development process to ensure that they are evaluating their DID method 
+      appropriately by the requirements as given by the problem they are attempting 
+      to solve with the introduction of a new DID method.
```
Comment (TallTed) [inline index.html:216]:
```suggestion
      DID method developers are encouraged to review and apply the
      [[DID-RUBRIC]] as early as possible in their development process to
      ensure that they are evaluating their DID method appropriately by the
      requirements of the problem they are attempting to solve with the
      introduction of a new DID method.
```

**Comment by mprorock** [comment]
thanks @TallTed

File: index.html [hunk 2]
```diff
@@ -210,10 +210,11 @@ <h2>Methods</h2>
     </p>
 
     <p class="advisement">
-      Developers are encouraged to utilize the [[DID-RUBRIC]] as early as poosible
-      in the development process to ensure that they are evaluating their DID method 
-      appropriately by the requirements as given by the problem they are attempting 
-      to solve with the introduction of a new DID method.
+      DID method developers are encouraged to review and apply the
+      [[DID-RUBRIC]] as early as possible in their development process to
+      ensure that they are evaluating their DID method appropriately by the
```
Comment (TallTed) [inline index.html:215]:
```suggestion
      ensure that they are evaluating their DID method appropriately for the
```
There's always another tweak...

Comment (mprorock) [reply index.html:215]: always!

**Review by OR13 — APPROVED** [review]

**Review by peacekeeper — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Review by rxgrant — APPROVED** [review]

**Pull request merged**